app/DashBoard/Presenters/templates/Blocks/favicons.latte

```diff
@@ -0,0 +1,8 @@
+{block favicon}
+<link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-touch-icon.png">
```

Comment:
Taky jsem o tom uvažoval, ale přišlo mi to jako overkill u takové věci. Funguje i bez basePath, taky jsem o tom zauvažoval.